front/src/modules/ui/action-bar/components/ActionBar.tsx

```diff
@@ -15,21 +15,46 @@ type StyledContainerProps = {
   position: PositionType;
 };
 
-const StyledContainer = styled.div<StyledContainerProps>`
+const StyledContainerContextMenu = styled.div<StyledContainerProps>`
```

Comment:
Let's create another component (ContextMenu) that's separate from ActionBar. We should also create ActionBarEntry and ContextMenuEntry types that should be similar